src/amo/components/SearchContextCard/index.js

```diff
@@ -1,95 +1,166 @@
-import PropTypes from 'prop-types';
+/* @flow */
 import * as React from 'react';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 
+import { fetchCategories } from 'core/reducers/categories';
 import translate from 'core/i18n/translate';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEMES_FILTER } from 'core/constants';
+import { withErrorHandler } from 'core/errorHandler';
 import Card from 'ui/components/Card';
-import {
-  ADDON_TYPE_EXTENSION,
-  ADDON_TYPE_DICT,
-  ADDON_TYPE_LANG,
-  ADDON_TYPE_THEME,
-} from 'core/constants';
+import type { AppState } from 'amo/store';
+import type { SearchFilters } from 'core/api/search';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { I18nType } from 'core/types/i18n';
+import type { DispatchFunc } from 'core/types/redux';
 
 import './styles.scss';
 
-export class SearchContextCardBase extends React.Component {
-  static propTypes = {
-    count: PropTypes.number,
-    filters: PropTypes.object,
-    i18n: PropTypes.object.isRequired,
-    loading: PropTypes.bool.isRequired,
-  };
+type Props = {||};
+
+type InternalProps = {|
+  ...Props,
+  categoryName?: string,
```

Comment:
it could also be undefined too so I guess i can either do 

`categoryName?: string | null`

or

`category: string | null | void`

I am not sure if 1  is better than the other..?

I wonder why flow didn't complain about this tho 🤔

Reply:
I just saw your comment below about defining this as null (which I will do:) thanks!